test/test_backend/routes/tst_graph.py

```diff
@@ -154,17 +154,29 @@ async def tst_get_connected_subgraph(self, only_fast_cases=True, save_output=Tru
                 # self.assertEqual(v, expected[k], msg=f'Results differ for {k} for test {test_name}')
                 assert v == expected[k], f'Results differ for "{k}" in test for "{test_name}"'
 
+    # noinspection PyUnboundLocalVariable removeNoinspectWhenMoreTestsAndMoveAssertsIntoLoop
     async def tst_concept_graph(self):
         """Test concept_graph()"""
         # Get results
         for test_type, test_name, codeset_ids, timeout_secs in [x.values() for x in self._get_test_cases()]:
             # todo: allow for more test cases, perhaps
             if test_name not in ['many-small']:
                 continue
-            sg, missing_in_betweens, hidden_dict, nonstandard_concepts_hidden = await concept_graph(codeset_ids)
+            sg: DiGraph
+            concepts: List[Dict[str, Any]]
+            hidden_by_voc: Dict[str, Set[int]]
+            nonstandard_concepts_hidden: Set
+            sg, missing_in_betweens, hidden_by_voc, nonstandard_concepts_hidden = await concept_graph(
+                codeset_ids,
+                # Hiding these in order to test functionality; nothing gets hidden w/ defaults
+                hide_vocabs=['SNOMED'], hide_nonstandard_concepts=True)
         # Assertions
-        # noinspection PyUnboundLocalVariable
-        assert len(sg.edges) > 1, 'Results differ'  # todo: create test cases
+        # self.assertEqual(...)
+        assert len(sg.nodes) == 14
+        assert len(sg.edges) == 24
+        assert len(missing_in_betweens) == 7
+        assert len(nonstandard_concepts_hidden) == 19
+        assert len(hidden_by_voc['SNOMED']) == 92
```

Comment:
## `test_concept_graph()`
Finished this test, and it is passing.

Could always use more test coverage, but right now I'm only testing `many-small`, and that's probably fine.

For this test, I turned `hide_nonstandard_concepts=True` and `hide_vocabs=['SNOMED']`. Nothing was hidden with defaults.